profile2/tool/src/java/org/sakaiproject/profile2/tool/pages/panels/MySocialNetworkingEdit.html

```diff
@@ -104,7 +104,12 @@
 	</form>
 	</div>
 	</div>
-
+	<script type="text/javascript">
+		$(document).ready(function(){
+			//initialize tooltip
+			$('[data-toggle="tooltip"]').tooltip();
+		});
+	</script> 
```

Comment:
Could this be added to `profile2/tool/src/webapp/javascript/profile2.js` or other instead of inline?

Reply:
This is in `profile2/tool/src/webapp/javascript/profile2.js` already. the problem is, that the tooltip element is created after the initialization. That is because it shows up after you click on edit at info / social networking. I don't know how to fix that, so i stuck to the same workaround... Do you know a better way?